google/cloud/bigtable/data/_async/client.py

```diff
@@ -393,8 +398,12 @@ def __init__(
         table_id: str,
         app_profile_id: str | None = None,
         *,
-        default_operation_timeout: float = 600,
-        default_per_request_timeout: float | None = None,
+        default_read_rows_operation_timeout: float = 600,
+        default_read_rows_attempt_timeout: float | None = None,
+        default_mutate_rows_operation_timeout: float = 600,
+        default_mutate_rows_attempt_timeout: float | None = None,
```

Comment:
We should have default for attempt timeout. mutate_rows = 1 minute, other operations = 20 seconds.

Reply:
Sure, should read_rows also be 20 seconds then?

Reply:
Also, previously we would raise an exception if the `attempt_timeout` was set greater than the `operation_timeout`, but I think that would be annoying if we have a default value pre-set, so I removed it. Let me know if you disagree with that